content/blog/how-to-open-new-file-in-new-tab-always-in-vscode/index.md

```diff
@@ -0,0 +1,23 @@
+---
+title: 'How to always keep open files in Visual Studio Code'
+date: "2020-01-18"
+description: Solution to open files always in a new tab in Visual Studio Code
+  - vscode
+  - tricks
+show: true
+author: shivam
+---
+
+
+Visual Code is am amazing editor and I sue it for my everyday code. One thing bug me everytime is the files are not open permanently until I use the shortcut `cmd + k` `enter` or right click and select `keep open` on the filename. This happens because VS Code opens the files by default in `preview mode`, so whenever the new file name is clicked from explorer, the opened file closes and new file is visible.
```

Comment:
```suggestion
Visual Code is am amazing editor and I use it for my everyday code. One thing bug me everytime is the files are not open permanently until I use the shortcut `cmd + k` `enter` or right click and select `keep open` on the filename. This happens because VS Code opens the files by default in `preview mode`, so whenever the new file name is clicked from explorer, the opened file closes and new file is visible.
```